base/src/main/java/proguard/evaluation/Variables.java

```diff
@@ -170,7 +171,7 @@ public Value getValue(int index)
         if (index < 0 ||
             index >= size)
         {
-            throw new IndexOutOfBoundsException("Variable index ["+index+"] out of bounds ["+size+"]");
+            throw new VariableInstructionIndexOutOfBoundException(index, size);
```

Comment:
Maybe just `VariableIndexOutOfBoundException`

Reply:
For consistency, I will update them all?